includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -660,7 +669,6 @@ public function using_proxy() {
 	 */
 	public function get_proxy_setup_url( $access_code = '', $error_code = '' ) {
 		$query_params = array(
-			'version'  => GOOGLESITEKIT_VERSION,
```

Comment:
We need to add `short_verification_token` to the proxy setup supports array as part of this PR if we remove `version` here. At the moment, the proxy back-fills that value based on the version, so as soon as we don't pass it anymore, we need to pass that value (just a reminder, this was the "feature" we added in 1.0.1 already, but back then it relied on the version since that was before we introduced `supports`).

Reply:
Added!